README.md

```diff
@@ -1,14 +1,21 @@
 # Supported tags and `Dockerfile` links
 
-- [`latest` (*Dockerfile*)](https://github.com/mono/docker/blob/master/3.12.0/Dockerfile)
-- [`onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/3.12.0/onbuild/Dockerfile)
-- [`3.12.0` (*Dockerfile*)](https://github.com/mono/docker/blob/master/3.12.0/Dockerfile)
-- [`3.12.0-onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/3.12.0/onbuild/Dockerfile)
+- [`latest` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.1.102/Dockerfile)
+- [`onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.1.102/onbuild/Dockerfile)
+- [`4.2.1.102` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.1.102/Dockerfile)
+- [`4.2.1.102-onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/4.2.1.102/onbuild/Dockerfile)
+- [`4.0.5.1` (*Dockerfile*)](https://github.com/mono/docker/blob/master/4.0.5.1/Dockerfile)
+- [`4.0.5.1-onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/4.0.5.1/onbuild/Dockerfile)
+- [`3.12.1` (*Dockerfile*)](https://github.com/mono/docker/blob/master/3.12.1/Dockerfile)
+- [`3.12.1-onbuild` (*onbuild/Dockerfile*)](https://github.com/mono/docker/blob/master/3.12.1/onbuild/Dockerfile)
```

Comment:
We should list 3.12.0 here too - even though it links to the same thing, some users might already be depending on the old tags